phpcs.xml.dist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset name="WPCLI-Vulnerability-Scanner">
+	<description>Custom ruleset for WPCLI-Vulnerability-Scanner</description>
+
+	<!-- What to scan. -->
+	<file>.</file>
+
+	<!-- Show progress. -->
+	<arg value="p"/>
+
+	<!-- Show standard. -->
+	<arg value="s"/>
+
+	<!-- Strip the filepaths down to the relevant bit. -->
+	<arg name="basepath" value="./"/>
+
+	<!-- Check up to 8 files simultaneously. -->
+	<arg name="parallel" value="8"/>
+
+	<!--
+	#############################################################################
+	USE THE WP_CLI_CS RULESET
+	#############################################################################
+	-->
+
+	<rule ref="WP_CLI_CS">
+		<exclude name="WordPress.NamingConventions.PrefixAllGlobals" />
+	</rule>
+	
+	<config name="testVersion" value="7.0-"/>
```

Comment:
We supposedly support [PHP 5.4+](https://github.com/10up/wpcli-vulnerability-scanner/blob/develop/wpcli-vulnerability-scanner.php#L8). I'd suggest bumping that version there instead of changing which version we're testing on here

Reply:
Yes, better to handle that in a separate follow-up PR, plugin's main file contains some PHPCS errors and warnings that should be fixed along with it.

Reply:
I have bumped PHP 7.0 and fixed PHPCS issues as well.